docs/Using-Fleet/Adding-hosts.md

```diff
@@ -257,10 +260,10 @@ Osqueryd will inherit the privileges from Orbit and does not need explicit permi
 #### Creating the profile
 Depending on your MDM, this might be possible in the UI or require a custom profile. If your MDM has a feature to configure *Policy Preferences*, follow these steps:
 
-1. Configure the identifier type to “path”
+1. Configure the identifier type to “path.”
 2. Paste the full path to Orbit as the identifier.
-3. Paste the full code signing identifier into the code requirement field.
-4. Allow “Access all files”. Access to Downloads, Documents etc is inherited from this.
+3. Paste the full code signing identifier into the code requirement field. 
```

Comment:
```suggestion
3. Paste the full code signing identifier into the Code requirement field. 
```
From what I can see capitalizing "Code" makes the most sense. It matches the UI and follows Fleet's sentence case rule.